packages/backend-deployer/src/cdk_deployer.ts

```diff
@@ -77,8 +78,7 @@ export class CDKDeployer implements BackendDeployer {
    * Wrapper for the child process executor. Helps in unit testing as node:test framework
    * doesn't have capabilities to mock exported functions like `execa` as of right now.
    */
-  executeChildProcess = async (
-    command: string,
+  executeChildProcessWithPackageManager = async (
```

Comment:
```suggestion
  executeCommand = async (
```
It's up to the implementation how that command gets executed

Reply:
Done a2f0e5ac84